docs/farming-&-staking/farming/advanced-cli/platforms/_linux.mdx

```diff
@@ -7,7 +7,7 @@ import styles from '@site/src/pages/index.module.css';
 ### Linux Filesystem Recommendations:
 
 :::note Important Notices for Linux Users
-- **SWAP**:  A swap is necessary only if your system has a low amount of RAM.
+- **SWAP**: A swap file is necessary only if your system has a low amount of RAM.
```

Comment:
Didn't know that on Linux you have to create a file for that, good idea to add this.